Gemfile

```diff
@@ -3,11 +3,3 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
-
-group :development do
-  gem 'rspec'
```

Comment:
Sorry, I accidentally merged. It's late at night here and I'm tired. I
added the last change to the Gemfile & .gemspec on master. Not how I wanted
to do it but it's working now. I don't mind you making other changes as
long as it's OK on CI.

BTW, I found CI failed once or twice due to a timing issue. You can see it
in the logs when that happens. There is a rerun button (I'm not sure if you
can re-run it, but you can let me know if it is an issue). All other
failures on CI that I saw were reporting problems with gems or ruby
versions.

Thanks,

Dave McNulla


On Fri, Apr 8, 2016 at 10:58 PM, Dave McNulla <mcnulla@gmail.com> wrote:

> The rspec has to stay with the cucumber or the tests will not run:
> https://travis-ci.org/dmcnulla/ntp/builds/121730524
>
> I can move the other to Gemfile.
>
> Thanks,
>
> Dave McNulla
>
>
> On Fri, Apr 8, 2016 at 10:52 PM, Павел Скрылёв (Malo Skrylevo) <
> notifications@github.com> wrote:
>
>> In Gemfile <https://github.com/dmcnulla/ntp/pull/1#discussion_r59110909>:
>>
>> > @@ -3,11 +3,3 @@ source 'https://rubygems.org'
>> >  gemspec
>> >
>> >  gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
>> > -
>> > -group :development do
>> > -  gem 'rspec'
>>
>> well the I believe pry, rubocop, and rspec can be moved back to Gemfile,
>> in order to not waste gemspec =)
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/dmcnulla/ntp/pull/1/files/24a259dc48a3962964a53e46b3778e6441c8ae2c..1ec80f8d6a1bf2887d99a8bd6f079fbd1167a85d#r59110909>
>>
>
>